packages/config/src/context.js

```diff
@@ -6,6 +6,31 @@ const { normalizeBuildCase } = require('./case')
 const { addCommandConfigOrigin } = require('./origin')
 const { deepMerge } = require('./utils/merge')
 
+// Takes a config object and adds each key to a namespace. This namespace is
+// usually `build`, since `context.*.{key}` is merged to `build.{key}`. The
+// only exception to that is for the `functions` key if the value is a plain
+// object, so that `context.*.functions.{key}` is merged to `functions.{key}`.
+const addNamespace = (configObject) => {
```

Comment:
`configObject` is `config.build`. Should the variable be named `build` instead to make it clearer?